tdrs-backend/docker-compose.yml

```diff
@@ -12,6 +12,15 @@ services:
       - POSTGRES_PASSWORD=something_secure
       - POSTGRES_DB=tdrs_test
       - POSTGRES_PORT=5432
+  localstack:
+    image: localstack/localstack:0.12.9
+    environment:
+      - SERVICES=s3
+      - DATA_DIR=/tmp/localstack/data
```

Comment:
We need to tell localstack to put data in the named volume we have created for it.